cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/tasks/CassandraState.java

```diff
@@ -640,6 +644,10 @@ public void update(Protos.TaskStatus status) throws IOException {
                     }
 
                     update(cassandraTask);
+                    if (cassandraTask.isTerminated()) {
+                        final String metricSuffix = this.getClass().getName() + "." + cassandraTask.getName();
```

Comment:
It will emit 1, per task failure. User has to aggregate count for a given time window to issue an alert. For example, if count = 3, for node-0 within 2 minutes, probably means a crash loop.